hack/_update-generated-proto-bindings-dockerized.sh

```diff
@@ -47,6 +47,10 @@ for api; do
 
     for file in "${protos[@]}"; do
         dir="$(dirname "${file}")"
-        kube::protoc::generate_proto "${dir}"
+        if [ "$dir" == staging/src/k8s.io/cri-api/pkg/apis/runtime/v1 ]; then
```

Comment:
special-casing this here is really weird... I'd actually love to see us switch *all* the protobinding generation from gogo to normal proto if possible